webui/react/src/pages/ModelDetails/ModelVersionActionDropdown.tsx

```diff
@@ -102,7 +67,8 @@ const ModelVersionActionDropdown: React.FC<Props> = (
         onVisibleChange={onVisibleChange}>
         {children}
       </Dropdown>
-      {contextHolder}
+      {modalModelDownloadContextHolder}
```

Comment:
do we want these to be at the parent level since we could have many action dropdowns rendering in one page at once?

Reply:
https://github.com/determined-ai/determined/pull/4468/commits/d85716d3df77a0c6e12ba99dd91e9496f71a06bb